robot_state state

---
uint32 error_code
